stories/forms/labels/labeled-fieldset.story.js

```diff
@@ -0,0 +1,113 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { LabeledFieldset, Checkbox as StaticCheckbox } from 'src'
+import dynamicInput from '../../dynamic-input'
+import { action } from '@storybook/addon-actions'
+
+const Checkbox = dynamicInput({
+  initialValue: false,
+  valuePath: 'input.value',
+  onChangePath: 'input.onChange',
+})(StaticCheckbox)
+
+const inputProps = {
+  name: 'person.option',
+  onChange: action('checkbox clicked'),
+}
+
+storiesOf('LabeledFieldset', module)
+  .add('default', () => (
+    <LabeledFieldset
+      {...{
+        input: {
+          name: 'inputName',
+        },
+        label: 'Select options',
+        meta: {},
+      }}
+    >
+      <Checkbox input={inputProps} meta={{}} />
+    </LabeledFieldset>
+  ))
+  .add('with error', () => (
+    <LabeledFieldset
+      {...{
+        input: {
+          name: 'inputName',
+        },
+        label: 'Select options',
+        meta: {
+          touched: true,
+          invalid: true,
+          error: 'Error message',
+        },
+      }}
+    >
+      <Checkbox input={inputProps} meta={{}} />
+    </LabeledFieldset>
+  ))
+  .add('hide error', () => (
+    <LabeledFieldset
+      {...{
+        input: {
+          name: 'inputName',
+        },
+        label: 'Select options',
+        meta: {
+          touched: true,
+          invalid: true,
+          error: 'Error message',
+        },
+        hideErrorLabel: true,
+      }}
+    >
+      <Checkbox input={inputProps} meta={{}} />
+    </LabeledFieldset>
+  ))
+  .add('with custom label', () => {
+    // eslint-disable-next-line
+    const CustomLabel = ({ onClickLabel }) => (
+      <legend onClick={onClickLabel} htmlFor="inputName">
```

Comment:
Can we pass a different custom prop (or use it in a different way)? I don't want to encourage inappropriate `onClick` handler usage